rt/rs/sse/src/main/java/org/apache/cxf/jaxrs/sse/OutboundSseEventImpl.java

```diff
@@ -24,24 +24,24 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.sse.OutboundSseEvent;
 
-public class OutboundSseEventImpl implements OutboundSseEvent {
-    private String id;
-    private String name;
-    private String comment;
-    private long reconnectDelay = -1;
-    private Class<?> type;
-    private Type genericType;
-    private MediaType mediaType;
-    private Object data;
+public final class OutboundSseEventImpl implements OutboundSseEvent {
+    private final String id;
+    private final String name;
+    private final String comment;
+    private final long reconnectDelay;
+    private final Class<?> type;
+    private final Type genericType;
+    private final MediaType mediaType;
+    private final Object data;
 
     public static class BuilderImpl implements Builder {
         private String id;
         private String name;
         private String comment;
         private long reconnectDelay = -1;
-        private Class<?> type;
+        private Class<?> type = String.class;
         private Type genericType;
-        private MediaType mediaType;
+        private MediaType mediaType = MediaType.SERVER_SENT_EVENTS_TYPE;
```

Comment:
I get NPEs when writing events via the `OutboundSseEventBodyWriter` if the user does not explicitly set the MediaType, so we will need some sort of default MediaType - the spec does not require the user to specify one.  `MediaType.SERVER_SENT_EVENTS_TYPE` (i.e. `text/event-stream`) seems to work for me, but I can try another type.  Would you suggest `text/plain`?